### ISSUE TITLE
Cannot generate listview using ignite-ir-next boilerplate commands

### ISSUE THREAD

**Issue by ali-hellani** [infinitered/ignite#988]
I have just installed **ignite 2** with the boilerplate **ir-next** then i wanted to generate a listview but i couldn't, here is the screenshot of `ignite generate` command (see the **command badfile description**):

<img width="697" alt="screen shot 2017-04-23 at 3 38 35 pm" src="https://cloud.githubusercontent.com/assets/10726596/25313687/2ac79a26-283c-11e7-83eb-90b5b79c1a4e.png">

---
`ignite doctor` results:

```
Alis-MacBook-Pro:Textables leolani$ ignite doctor
System
  platform           darwin
  arch               x64
  cpu                8 cores                                               Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  directory          /Users/leolani/code/reactnative-workspace/Textables

JavaScript
  node               7.9.0        /usr/local/bin/node
  npm                4.6.1        /usr/local/bin/npm
  yarn               0.23.2       /usr/local/bin/yarn

React Native
  react-native-cli   2.0.1
  app rn version     0.42.0

Ignite
  ignite             2.0.0-beta.8   /usr/local/bin/ignite

Android
  java               1.8.0_101    /usr/bin/java
  android home       -            /Users/leolani/Library/Android/sdk

iOS
  xcode              8.2.1

```

**Comment by ali-hellani** [comment]
I have realised that each generator file that has the line of code below doesn't work:
`const patterns = require('../lib/patterns');`

There is no such directory with this name.

**Comment by skellock** [comment]
Thanks for the heads up @ali-hellani.  This is now fixed in `ignite-ir-next@0.0.7`.  Bumping your devDependency in your package.json will pick up this fix.

**Comment by ali-hellani** [comment]
it worked, thanks :)
